crates/ra_ide_api/src/change.rs

```diff
@@ -43,7 +43,7 @@ impl fmt::Debug for AnalysisChange {
         if !self.libraries_added.is_empty() {
             d.field("libraries_added", &self.libraries_added.len());
         }
-        if !self.crate_graph.is_some() {
```

Comment:
Heh, this is outright a bug, it should be `!self.crate_graph.is_none()`. Good clippy!